examples/protonect/src/shader/stage1.fs

```diff
@@ -106,6 +104,9 @@ void main(void)
   vec2 ab1 = processMeasurementTriple(uv, P0Table1, 3, Params.ab_multiplier_per_frq.y, saturated.y);
   vec2 ab2 = processMeasurementTriple(uv, P0Table2, 6, Params.ab_multiplier_per_frq.z, saturated.z);
   
+#if MESA_BUGGY_BOOL_CMP
```

Comment:
change this to #ifdef and then simply prepend #define MESA_BUGGY_BOOL_CMP to every source file?